ownCloud Intents/CreateFolderIntentHandler.swift

```diff
@@ -103,13 +150,26 @@ public class CreateFolderIntentHandler: NSObject, CreateFolderIntentHandling {
 		completion(OCBookmarkManager.shared.accountList, nil)
 	}
 
+	@available(iOSApplicationExtension 14.0, *)
```

Comment:
In my opinion this code is available since iOS 13.0

Reply:
At least in the headers, `INObjectCollection` is marked as available only as of iOS 14:
<img width="766" alt="Bildschirmfoto 2021-03-29 um 23 04 35" src="https://user-images.githubusercontent.com/639669/112900001-2359dd80-90e3-11eb-83dd-294750326723.png">